components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/impl/DeviceDAOImpl.java

```diff
@@ -291,6 +303,72 @@ public Device getDevice(int deviceId, int tenantId) throws DeviceManagementDAOEx
         return devices;
     }
 
+    @Override
```

Comment:
It's always better to add a meaningful comment for public methods that we are implementing.